### ISSUE TITLE
KeyError while converting pytorch model to IR: no onnx::Sigmoid and onnx::Mul operations in pytorch_parser

### ISSUE THREAD

**Issue by slomrafgrav** [Microsoft/MMdnn#509]
Platform (like ubuntu 16.04/win10): ubuntu 18.04

Python version: 3.6.7

Source framework with version (like Tensorflow 1.4.1 with GPU): pytorch 0.4.0

Destination framework with version (like CNTK 2.3 with GPU): mmdnn 0.2.2 

Pre-trained model path (webpath or webdisk path): https://github.com/Cadene/pretrained-models.pytorch/blob/master/pretrainedmodels/models/senet.py

Running scripts:
from mmdnn.conversion.pytorch.pytorch_parser import PytorchParser 
parser = PytorchParser('model.pth',[3,224,224])
parser.run('sen')

For both sigmoid module and multiplication we get error, that it is KeyError, because this operations are not implemented in pytorch_parser
Addition of rename_Sigmoid and rename_Mul methods is needed

**Comment by rainLiuplus** [comment]
Thanks for your contribution!
